src/Dispatcher.cpp

```diff
@@ -264,10 +269,10 @@ void DispatcherBase::DispatchThreadMain() {
         // post outgoing messages if connection is active
         // only send keep-alives on client
         if (conn->state() == NetworkConnection::kActive)
-          conn->PostOutgoing(!m_server);
+          conn->PostOutgoing(m_networkMode & NT_NET_MODE_CLIENT);
```

Comment:
Well before that would have been the behavior, since you could have m_server and !m_active. On the topic of that, I'd have to look through the code more, but what about m_active being changed to that flag? that way theyre never out of sync. I'd definitely need @PeterJohnson opinion on that one though.